cmp/grid/Grid.js

```diff
@@ -378,9 +376,9 @@ class LocalModel {
             {agGridModel, store, experimental} = model;
 
         return {
-            track: () => [agGridModel.agApi, this.frameworkCmpsMounted, store.lastLoaded, store.lastUpdated, store._filtered, model.showSummary],
-            run: ([api, frameworkCmpsMounted, lastLoaded, lastUpdated, newRs]) => {
-                if (!api || !frameworkCmpsMounted) return;
+            track: () => [agGridModel.agApi, agGridModel.agColumnApi, this.frameworkCmpsMounted, store.lastLoaded, store.lastUpdated, store._filtered, model.showSummary],
+            run: ([agApi, colApi, frameworkCmpsMounted, lastLoaded, lastUpdated, newRs]) => {
+                if (!agApi || !frameworkCmpsMounted) return;
```

Comment:
I would add the explicit check here that colApi is also available.  It seems like you aren't sure that it will be from the check on line 401, which should be removed.

Reply:
otherwise you are just opening us up for re-executing this code multiple times.

Reply:
Done.